atsign-foundation/at_c
# feat: add test for RSA 4096 operability

**- What I did**
Added test that verifies at_c's capability of withstanding a 512-byte key

**- How I did it**
Updated verify func to determine if 256 or 512-byte key is used, created a test that uses a 512-byte generated public key

**- How to verify it**
Run the tests and verify that tests pass with
```
cmake -S . -B build -DATSDK_BUILD_TESTS="unit" && cmake --build build --target all && ./build/packages/atchops/tests/test_rsa_verify
```

**- Description for the changelog**
feat: tests for RSA 4096
